dev/com.ibm.ws.crypto.passwordutil/src/com/ibm/websphere/crypto/PasswordUtil.java

```diff
@@ -266,7 +266,6 @@ public static String encode(String decoded_string, String crypto_algorithm,
             decoded_string = decoded_string.trim();
         }
         String encoded_string = encode_password(decoded_string, crypto_algorithm.trim(), properties);
-
```

Comment:
we can remove this file from the commit, so the history will be cleaner on this file

Reply:
Fixed in https://github.com/OpenLiberty/open-liberty/pull/30769/commits/d1b8ad9b6d92657b2e65bb77c2d7db15790cdd57